### PR TITLE
Add headers list

### PR DESCRIPTION
**Pull request by roxblnfk**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #5

Some experimental and deprecated headers i did not add

### CODE REVIEW

File: src/Header.php [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+    public const X_DNS_PREFETCH_CONTROL = 'X-DNS-Prefetch-Control';
+
+    /**
+     * Pagination
+     */
+
+    /**
+     * The per page limit.
+     */
+    public const X_PAGINATION_LIMIT = 'X-Pagination-Limit';
```
Comment (xepozz) [inline src/Header.php:669]: I don't sure that these pagination headers should be added.
These not declared in any HTTP standarts. Could you delete?

Comment (samdark) [reply src/Header.php:669]: These are OK since these are commonly used in APIs.

File: src/Header.php [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
+namespace Yiisoft\Http;
+
+/**
+ * HTTP headers
+ * @link https://developer.mozilla.org/docs/Web/HTTP/Headers
+ */
+final class Header
+{
+    /**
+     * Authentication
```
Comment (samdark) [inline src/Header.php]: Docblock should be for something but this one is not attached to any element. Same for other sections.

Either it should be just block comment such as `/* */` or inline comment such as `//` or should be removed.

**Comment by samdark** [comment]
Looks OK overall but let's fix dockblocks.

**Review by samdark — APPROVED** [review]

**Pull request merged**